scenarios/centos-9/ceph_backends.yml

```diff
@@ -30,7 +30,9 @@ cifmw_run_tests: true
 cifmw_tempest_default_groups: &test_operator_list
   - openstack-operator
   - cinder-operator
+  - ironic-operator
   - keystone-operator
+  - manila-operator
```

Comment:
if you hit any tempest test failure because of missing share type 'default', you will need to create this share type in advance[1], maybe as a pre hook in [2]

[1] https://steps.ci.openshift.org/reference/openstack-k8s-operators-tempest#line71
[2] https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/08-run-tests.yml#L1-L5